entities-search/src/main/scala/io/renku/entities/searchgraphs/commands/SearchInfoFetcher.scala

```diff
@@ -106,14 +106,16 @@ private class SearchInfoFetcherImpl[F[_]: Async: Logger: SparqlQueryTimeRecorder
         |""".stripMargin
   )
 
+  private lazy val rowsSeparator = "##"
```

Comment:
Maybe we could use the null byte? But not sure if jena/sparql supports this. This would be even harder to type in for users 😄 Just thinking when the first name with a double hash occurs 😰

Reply:
Yes, that's definitely a stronger choice 👍🏻